submissions/ORybka/post-a-tiny-js-world/index.js

```diff
@@ -0,0 +1,65 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/ORybka/a-tiny-JS-world
+   Web app: https://orybka.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+  constructor(species, name, gender, legs, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  callFriends() {
+    return this.friends.join(', ');
+  }
+
+  getValues(addValue) {
+    return [this.species, `<strong>${this.name}</strong>`, this.gender, this.legs, addValue, `<em>${this.saying}</em>`, this.callFriends()];
+  }
+
+  printValues() {
+    return this.getValues().join('; ');
+  }
+}
+
+class Human extends Inhabitant {
+  constructor(name, gender, saying, friends) {
+    super('human', name, gender, 2, saying, friends);
+    this.hands = 2;
+  }
+
+  getValues() {
+    return super.getValues(this.hands);
+  }
```

Comment:
```suggestion
  getValues() {
    return [...super.getValues, this.hands];
  }
```

Reply:
Ah! Spread operator! Of course! And function as an argument
@OleksiyRudenko  Thank you very much for the hint. This is a great solution!
I will use it now 😄

Reply:
@ORybka There was a typo. Not as agrument, but rather a normal function call just as it is now in the code suggestion

Reply:
Yes, function call, thanks :)